Source/UI/NewUI/BillingForm/View/Field/BillingFormTextField.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+class BillingFormTextField: UITextField {
```

Comment:
Question : I know we are planning to reuse some parts of the Billing components inside Card Number field, Expiry and CVV to make our life's easier and get other components developed quickly. But with the existing structure, will we be able to reuse the code and extend it to other components ?

Reply:
Updated